source/getting-started-edge/index.rst

```diff
@@ -0,0 +1,245 @@
+
+.. _InCommon: https://incommon.org/federation
+
+.. _getting-started:
+
+=============================
+Getting started with CHI@EDGE
+=============================
+
+This guide will walk you through the initial steps of getting an account,
+joining a project and working with your first container.
+
+.. contents:: :local:
+
+.. _getting-started-user:
+
+Step 1: Log in to Chameleon
+===========================
+
+Just click the “Log in” button situated in the top right corner of `our main page
```

Comment:
Do these smart quotes render OK?

Reply:
(I'm guessing this was copied, so it must!)

Reply:
They render as the same character smart quotes, nothing is converted.